.gitignore

```diff
@@ -17,6 +17,9 @@ contribution/vagrant/*.log
 *.rpm
 .coverprofile
 
+#Object files
+*.o
```

Comment:
@daemon1024 We should also remove [these](https://github.com/kubearmor/KubeArmor/tree/main/KubeArmor/enforcer/bpflsm) object files from the tree, right?

Reply:
Those are needed though, since they cannot be generated at runtime in GH Action (due to lack of BPFLSM support) . Thanks for noticing, I will update the ignore to be `BPF/*.o`

Reply:
The object files in enforcer/bpflsm follow CO-RE ( Compile Once Run Everywhere ) hence it's fine for them to be in the tree, this is not the case for monitor code. Hence the need to clean them out.

Reply:
Gotcha :+1: